ReactiveFeedback/Feedback.swift

```diff
@@ -80,22 +80,23 @@ public struct Feedback<State, Event> {
     }
 
     /// Creates a Feedback which re-evaluates the given effect every time the
-    /// given predicate passes.
+    /// predicate passes a new occurrence of `State` (i.e. every state change).
     ///
     /// If the previous effect is still alive when a new one is about to start,
-    /// the previous one would automatically be cancelled.
+    /// or if the predicate evaluates to `false`, the previous one would automatically
+    /// be cancelled.
     ///
     /// - parameters:
     ///   - predicate: The predicate to apply on the state.
     ///   - effects: The side effect accepting the state and yielding events
     ///              that eventually affect the state.
     public init<Effect: SignalProducerConvertible>(
-        predicate: @escaping (State) -> Bool,
+        occurrencesPassing predicate: @escaping (State) -> Bool,
```

Comment:
I'm not convinced that this name change makes the behaviour any more obvious TBH, and the name just sounds odd. Maybe `everyTime` would make this clearer?

Reply:
I will have to agree with @mluisbrown on this one. Unfortunately, at this point in time, I cannot provide a much better alternative besides replacing `ocurrences` with `events` or `state`.

Reply:
I agree with @mluisbrown, the name doesn't sound familiar and doesn't make it obvious IMO... Why not using simply `filter`? Because that aligns with the operator that we have in signals which don't apply any logic other than evaluating if the value should be filtered out or not, if we want to skip repeated values we always need to apply `skipRepeats` so kind of seems aligned. But `everyTime` seems a good alternative too.

Reply:
What about adding an argument label to the 2nd parameter? Like `filter:whenPassing:`?

Reply:
You mean something like `filter: (State) -> Bool, skippingRepeated: Bool` to merge both overloads? Not sure if that's you meant but I love that idea and definitely makes this more intuitive IMO.

Reply:
I would not recommend "merging them" because they are different enough.

To be honest, I now start to think we should nuke this variant, because in practice it rarely makes sense. We should replace it with a "when predicate outcome has flipped" behaviour, which starts & cancels the effect based on transitions in the boolean output.

Reply:
The close analogy in electronics is edge triggered flip flops, as a naming inspiration :)

Reply:
BTW it seems [RxFeedback](https://github.com/NoTests/RxFeedback.swift/tree/master/Sources/RxFeedback) dodged this problem by forcing everyone to create `Equatable` nominal request types. 😃

Reply:
> We should replace it with a "when predicate outcome has flipped" behaviour

I love this 👍